docs/kubefirst/local/limitations.md

```diff
@@ -0,0 +1,32 @@
+# Limitations running Kubefirst Locally 
+
+## Overview
+
+`Kubefirst local` bring to our local machine the experience of having a cloud environment running locally. This is the best effort to abstract all the convenience of cloud services running inside containers. 
+Thank you for all the projects listed in the [Credit](../local/credit.md) section. 
+
+Unfortunately, we have some limitations running kubefirst locally, and we listed below these limitations and possible ways to solve them.
+
+## Hardware recommendation
+
+- OS: macOS (Intel or Apple Silicon M1/M2), Linux AMD64, Windows 10 (Ubuntu WSL)
+- CPU: A Quad Core CPU or Apple Silicon with M1 or M2 chip
+- RAM: 16GB RAM
```

Comment:
out of curiosity, where did we get the value from?

Reply:
The amount of CPU/RAM I am putting in is based on tests I ran on an old MacBook Pro with similar specs.
We validate M1 and M2 by running them on a daily basis.
Linux was validated by @marinhojess during their onboarding and by @6za with the container's environment.
I just checked with @marinhojess, and the Windows 10 with Ubuntu in WSL for local didn't work as expected, only for the cloud flavors. So I will remove the mention since this doc is exclusive to local installs...

Reply:
awesome! I also tested windows this morning, there is an issue with disk validation, then I realised we dont support windows.

Reply:
thank you for reviewing